cscs-checks/system/jobreport/gpu_report.py

```diff
@@ -23,14 +23,11 @@ class gpu_usage_report_check(GpuBurn):
     '''
 
     valid_systems = ['daint:gpu', 'dom:gpu']
-    valid_prog_environs = ['PrgEnv-gnu']
+    valid_prog_environs = ['PrgEnv-nvidia']
```

Comment:
There is no need to redefine this, since the  `gpu_burn_check` is now using `+cuda` or `+hip`. Scoping the systems makes sense though, since job report is available only on those two.